src/mapeo-manager.js

```diff
@@ -142,4 +144,23 @@ export class MapeoManager {
 
     return project
   }
+
+  /**
+   * @returns {Promise<Array<Pick<ProjectValue, 'name'> & { projectId: string, createdAt: string, updatedAt: string }>>}
+   */
+  async listProjects() {
+    return this.#db
+      .select({
+        projectId: projectTable.docId,
+        createdAt: projectTable.createdAt,
+        updatedAt: projectTable.updatedAt,
+        name: projectTable.name,
+      })
+      .from(projectTable)
+      .all()
+      .map((value) => ({
```

Comment:
Mapbyr use the denullify function here?

Reply:
addressed via https://github.com/digidem/mapeo-core-next/pull/208/commits/5ff8aad4bd4b8f86bfc76957b27ab1cde0f1b8ec